commands/developer-utils/GitHub-PRs-Inline.rb

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open Pull Requests
+# @raycast.mode inline
+# @raycast.refreshTime 5m
+# @raycast.packageName GitHub
+#
+# Optional parameters:
+# @raycast.iconDark images/github-logo-iconDark.png
+# @raycast.icon images/github-logo.png
+# @raycast.needsConfirmation false
+
+require 'json'
+require 'net/http'
+require 'uri'
+
+API_TOKEN = ''
+RED = 31
+GREEN = 32
+YELLOW = 33
+
+
+if API_TOKEN.empty?
+  puts 'No API token provided'
+  exit(1)
+end
+
+uri = URI('https://api.github.com/graphql')
+req = Net::HTTP::Post.new(uri)
+req['Authorization'] = "token #{API_TOKEN}"
+req.body = '{ "query": "query { viewer { pullRequests(first: 10, states:OPEN) { nodes { title number url } }}}" }'
+req_options = {
+  use_ssl: uri.scheme == 'https'
+}
+
+res = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
+  http.request(req)
+end
+
+if res.code == '200'
+  result = JSON.parse(res.body)
+  pull_requests = result['data']['viewer']['pullRequests']['nodes']
+
+  if pull_requests.length == 0
+    MESSAGE = "No open pull requests 🎉"
+    puts "\e[#{GREEN}m#{MESSAGE}\e[0m"
+  elsif pull_requests.length <= 5
+    MESSAGE = "You have #{pull_requests.length} open pull requests"
+    puts "\e[#{YELLOW}m#{MESSAGE}\e[0m"
+    pull_requests.each do |pr|
+      puts "##{pr['number']} #{pr['title']} (#{pr['url']})"
+    end
+  elsif pull_requests.length > 5
+    MESSAGE = "You have #{pull_requests.length} open pull requests"
+    puts "\e[#{RED}m#{MESSAGE}\e[0m"
+    pull_requests.each do |pr|
+      puts "##{pr['number']} #{pr['title']} (#{pr['url']})"
+    end
+  end
```

Comment:
I think you can avoid some duplicated code here.

```suggestion
  else
    color = (pull_requests.length <= 5) ? YELLOW : GREEN
    message = "You have #{pull_requests.length} open pull requests"
    puts "\e[#{color}m#{message}\e[0m"
    pull_requests.each do |pr|
      puts "##{pr['number']} #{pr['title']} (#{pr['url']})"
    end
  end
```